### DISCUSSION TITLE
does it work on Windows?

### DISCUSSION THREAD

**Discussion by insinfo** [dropbox/fast_rsync#16] [General]
I developed a small linux server backup application in flutter/dart that runs on a local windows machine, this application backs up servers via SSH/SFTP, but currently it only does full backups, it doesn't do incremental backups and it's quite slow, I I want to port this application to rust, using the lib "iron-carrier" makes the synchronization of a directory on a linux server to a windows machine?

https://github.com/insinfo/fsbackup

**Comment by goffrie** [comment]
`fast_rsync` does work on Windows. It's portable because it doesn't access the filesystem or interface with the operating system in any significant way - that would be your job as a library user.
